pkg/distro/fedora/images.go

```diff
@@ -174,6 +158,49 @@ func osCustomizations(
 		osc.YUMRepos = append(osc.YUMRepos, osbuild.NewYumReposStageOptions(filename, repos))
 	}
 
+	if oscapConfig := c.GetOpenSCAP(); oscapConfig != nil {
+		if t.rpmOstree {
+			panic("unexpected oscap options for ostree image type")
+		}
+		var datastream = oscapConfig.DataStream
+		if datastream == "" {
+			datastream = oscap.DefaultFedoraDatastream()
+		}
+
+		oscapStageOptions := osbuild.OscapConfig{
+			Datastream: datastream,
+			ProfileID:  oscapConfig.ProfileID,
+		}
+
+		if oscapConfig.Tailoring != nil {
+			newProfile, tailoringFilepath, tailoringDir, err := oscap.GetAutotailorFile(oscapConfig.ProfileID)
+			if err != nil {
+				panic("unexpected error creating autotailoring file options")
```

Comment:
It'd be good to print the error (`err`) here in the panic message for easier troubleshooting.